revolv/static/project.scss

```diff
@@ -43,6 +43,439 @@
     }
 }
 
+body {
+    font-family: Lato;
+}
+
+.row {
```

Comment:
yeah! Foundation by default adds a margin to the rows so I wanted to make them fullscreen.